projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -145,7 +144,8 @@ export default function Edit( { attributes, setAttributes, clientId } ) {
 		[ categories ]
 	);
 
-	const tags = useSelect( select => select( 'core/editor' ).getEditedPostAttribute( 'tags' ), [] );
+	const tags =
+		useSelect( select => select( 'core/editor' ).getEditedPostAttribute( 'tags' ), [] ) || [];
```

Comment:
```suggestion
		useSelect( select => select( 'core/editor' ).getEditedPostAttribute( 'tags' ) ) || [];
```
I'll test, but I'm pretty sure because of this it's only running once.  I'm happy to be wrong.

Reply:
OK, looks good after testing.  But apparently I still don't understand how the useSelect hook works...